test/common.py

```diff
@@ -1,5 +1,4 @@
 import pytest  # noqa
-from pytest import skip, raises  # noqa
```

Comment:
The libraries were imported and not used

<img width="941" alt="image" src="https://user-images.githubusercontent.com/3242828/235293547-765007ea-363d-4cfd-85c5-bc9a4fe0f47b.png">